js/interface.js

```diff
@@ -2151,7 +2151,14 @@ $('#show-access-rules').click(function() {
       cursor: '-webkit-grabbing; -moz-grabbing;',
       axis: 'y',
       forcePlaceholderSize: true,
+      forceHelperSize: true,
       revert: 150,
+      helper: function(event, row) {
+        row.children().each(function() {
+          $(this).width($(this).width());
+        });
+        return row;
```

Comment:
Empty Line added. ESLint is not capturing this error.

Reply:
@NikitaJadhavXoriant Thanks for checking. Looks like it's because [the file uses ES6 syntax](https://github.com/Fliplet/fliplet-widget-data-source/blob/f39e134ecf6619f985044591bf17beec654fcbb3/js/interface.js#L344) while the ESLint only allows ES5. We'll sort it out separately.